### PR TITLE
DEV 2094 drop py36

### PR DESCRIPTION
**Pull request by qiaouchicago**

### CODE REVIEW

**Review by BenX00 — APPROVED** [review]

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -12,10 +12,10 @@ tox:
   stage: test
   parallel:
     matrix:
-      - PYTHON_VERSION: ['3.6', '3.7', '3.8']
+      - PYTHON_VERSION: ['3.7', '3.8']
   image: ${BASE_CONTAINER_REGISTRY}/python${PYTHON_VERSION}-builder:${BASE_CONTAINER_VERSION}
   services:
-    - name: docker.osdc.io/ncigdc/ci-postgres-13:feat_dev-1924-postgres-init-script-3673ef90
+    - name: docker.osdc.io/ncigdc/ci-postgres-13:2.3.2
```
Comment (wrkhenddher) [inline .gitlab-ci.yml:18]: 😵‍💫

**Review by XukunXiangUChicago — APPROVED** [review]

File: dev-requirements.txt
Comment (wrkhenddher) [inline dev-requirements.txt:1]: https://www.youtube.com/watch?v=Eo-KmOd3i7s&t=65s

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+source = ["gdcdatamodel"]
+
+[tool.coverage.html]
+title = "gdcdatamodel coverage report"
+directory = "htmlcov"
+show_contexts = true
+
+[tool.isort]
+profile = "black"
+known_third_party = ["addict", "authutils", "cdiserrors", "cdislogging", "ciso8601", "deepdiff", "defusedcsv", "dotenv", "envelopes", "flask", "flask_cors", "gdc_ng_models", "gdcdictionary", "httmock", "indexclient", "indexd", "indexd_test_utils", "jsonschema", "mock", "pkg_resources", "psqlgraph", "pytest", "requests", "setuptools", "simplejson", "six", "sqlalchemy", "yaml"]
```
Comment (wrkhenddher) [inline pyproject.toml:32]: What's this for? Is it needed?
Just wondering about keeping these updated.

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+known_first_party = ["gdcdatamodel", "setup_batch", "setup_notifications", "setup_psqlgraph", "setup_redactionslog", "setup_report", "setup_study_data", "setup_transactionlogs"]
```
Comment (wrkhenddher) [inline pyproject.toml:33]: Likewise.

File: requirements.txt
Comment (wrkhenddher) [inline requirements.txt:1]: https://www.youtube.com/watch?v=Eo-KmOd3i7s&t=65s

**Review by wrkhenddher — APPROVED** [review]

File: pyproject.toml (hunk 2 above)
Comment (qiaouchicago) [reply pyproject.toml:32]: for isort to know how to automatically sort imports. 
Even thought we do not have isort for this repo yet. 
I think isort was removed from pre-commit-config for some release.

File: dev-requirements.txt
Comment (qiaouchicago) [reply dev-requirements.txt:1]: LOL, I could not remember how many times I have listened this song when it came out. It was very popular in China.

**Pull request merged**